plasmapy/physics/parameters.py

```diff
@@ -1116,11 +1135,150 @@ def Debye_number(T_e, n_e):
     """
 
     lambda_D = Debye_length(T_e, n_e)
+
     N_D = (4 / 3) * np.pi * n_e * lambda_D ** 3
 
     return N_D.to(u.dimensionless_unscaled)
 
 
+@utils.check_quantity({
+    'V_0': {'units': u.V, 'can_be_negative': False},
+    'n_s': {'units': u.m ** -3, 'can_be_negative': False}
+    })
+def matrix_sheath_thickness(V_0, n_s):
+    r"""Calculate the matrix sheath thickness for a high potential sheath.
+
+    Parameters
+    ----------
+    V_0 : ~astropy.units.Quantity
+        Sheath potential in units convertible to V.
+
+    n_s : ~astropy.units.Quantity
+        Sheath ion density in units convertible to m**-3.
+
+    Returns
+    -------
+    s : ~astropy.units.Quantity
+        Matrix sheath thickness in meters.
+
+    Raises
+    ------
+    TypeError
+        If one or more of the supplied parameters are not of the correct type.
+
+    ~astropy.units.UnitConversionError
+        If one or more parameter cannot be succesfully converted to the
+        respective unit.
+
+    ValueError
+        If one or more parameter does not have an appropriate value.
+
+    Warns
+    -----
+    ~astropy.units.UnitsWarning
+        If units are not provided, SI units are assumed.
+
+    Notes
+    -----
+    The matrix sheath model assumes an uniform ion sheath density and a large
+    sheath potential compared to the electron temperature. The sheath
+    thickness is given by
+
+    .. math::
+        s = \sqrt{ \frac{2 \epsilon_0 V_0}{e n_s} }.
+
+    See Also
+    --------
+    Child_law_sheath_thickness
+
+    Example
+    -------
+    >>> from astropy import units as u
+    >>> matrix_sheath_thickness(80 * u.V, 1e18 * u.m**-3)
+    <Quantity 9.40327552e-05 m>
+
+    """
+
+    s = np.sqrt(2 * eps0 * V_0 / (e * n_s))
+
+    return s.to(u.m)
+
+
+@utils.check_quantity({
+    'T_e': {'units': u.K, 'can_be_negative': False},
+    'V_0': {'units': u.V, 'can_be_negative': False},
+    'n_s': {'units': u.m ** -3, 'can_be_negative': False}
+    })
+def Child_law_sheath_thickness(T_e, V_0, n_s):
+    r"""Calculate the Child law sheath thickness.
+
+    Parameters
+    ----------
+    T_e : ~astropy.units.Quantity
+        Electron temperature in units convertible to K.
+
+    V_0 : ~astropy.units.Quantity
+        Sheath potential in units convertible to V.
+
+    n_s : ~astropy.units.Quantity
+        Sheath ion density in units convertible to m**-3.
```

Comment:
Might be good to include some more description here of exactly which density is meant, since in the Child-law sheath the ion density is changing throughout the sheath.  I'm 95% sure here you mean the density at the "sheath edge", or the point at which the ions have been accelerated to the Bohm speed through the presheath, and quasineutrality begins to break down. I think for a collisionless presheath this density is 1/sqrt(e) n_i ~= 0.606 n_i of the "bulk" plasma density n_i outside the presheath.

Reply:
Yes, and make sure to make this clear throughout the Langmuir probe code.